### PR TITLE
updates of caja, mate_common, mate_desktop

### LINKED ISSUE #8623: (might be very serious) Mismatched checksum for caja package
Attempting to install the caja package returns a mismatched checksum. This may or may not be very serious, but either way the installation fails with this error message:

```
crew install caja
caja: Caja file manager for the MATE desktop
https://mate-desktop.org
Version: 1.27.0
License: GPL-2+ LGPL-2+
Performing pre-flight checks...
No precompiled binary available for your platform, downloading source...
Checksum mismatch :/ Try again?

                      Filename: caja-1.27.0.tar.xz
    Expected checksum (SHA256): 82aa0cc7a50a844d686ac3d29ca1866033ba4fb0c03e198838161337301de6d5
  Calculated checksum (SHA256): afdbd0ca7f06c84777124793b388ae6ab8c34acb7d34ab7cdbae47e126d883bb
```

System information:

<details><summary>Expand</summary>

- Architecture: `x86_64` (`x86_64`)
- Processor vendor: `GenuineIntel`
- User space: `64-bit`
- Chromebrew Kernel version: `5.4`
- Chromebrew Running in Container: `false`

- Chromebrew version: `1.35.6`
- Chromebrew prefix: `/usr/local`
- Chromebrew libdir: `/usr/local/lib64`

- Last update in local repository: d4ed8038 `Brave 1.57.57 => 1.57.62 (#8620) (21 hours ago)`

- OS variant: `Chrome OS`
- OS version: `dedede-release/R116-15509.72.0`
- OS channel: `stable-channel`

</details>

### PR DESCRIPTION
**Pull request by satmandu**
Fixes #8623

Builds properly:
- [x] `x86_64`
- [x] `armv7l` 

### Run the following to get this pull request's changes locally for testing.
```
CREW_TESTING_REPO=https://github.com/satmandu/chromebrew.git CREW_TESTING_BRANCH=caja CREW_TESTING=1 crew update
```

### CODE REVIEW

File: packages/caja.rb [hunk 1]
```diff
@@ -1,45 +1,61 @@
...
+  depends_on 'libxcb' # R
+  depends_on 'libxdmcp' # R
+  depends_on 'libxext' # R
+  depends_on 'libxml2' # R
+  depends_on 'libxrandr' # R
+  depends_on 'libxrender' # R
+  depends_on 'mate_desktop' # R
+  depends_on 'pango' # R
+  depends_on 'util_linux' # R
+  depends_on 'zlibpkg' # R
```
Comment (uberhacker) [inline packages/caja.rb:56]: Did `run_tests` fail?

File: packages/mate_common.rb [hunk 2]
```diff
@@ -1,36 +1,24 @@
...
-    aarch64: 'd8e420843837ccdb2883abd53c46667444499db3cf041a907af29d4bed435e1d',
-     armv7l: 'd8e420843837ccdb2883abd53c46667444499db3cf041a907af29d4bed435e1d',
-       i686: 'f7a9d18c6c0a7f10828d0bc4528dd0b67405e0388287123516b04e6d80361223',
-     x86_64: 'd04c069aee911be73e5c80675e15a6bdfe3381432fc8bf293d6c29a592412694'
+    aarch64: '368b90c8aaaee44136b49fe764dfd2f348a82c0c97ec7f9d6a2e7b9f58ab764d',
+     armv7l: '368b90c8aaaee44136b49fe764dfd2f348a82c0c97ec7f9d6a2e7b9f58ab764d',
+     x86_64: '6b2311710ffa148aa1ae51212f3cd107dd0fec6584371e887e1f086e1f3a04ec'
   })
 
   depends_on 'gtk_doc'
```
Comment (uberhacker) [inline packages/mate_common.rb:23]: Did `run_tests` succeed?

File: packages/caja.rb (hunk 1 above)
Comment (satmandu) [reply packages/caja.rb:56]: I don't see any tests being run.

Should we make build systems/autotools run tests automatically?

File: packages/mate_desktop.rb [hunk 3]
```diff
@@ -1,38 +1,42 @@
...
+  depends_on 'libxau' # R
+  depends_on 'libxcb' # R
+  depends_on 'libxdmcp' # R
+  depends_on 'libxext' # R
+  depends_on 'libxrandr' # R
+  depends_on 'libxrender' # R
+  depends_on 'mate_common' => :build
+  depends_on 'pango' # R
+  depends_on 'zlibpkg' # R
 end
```
Comment (uberhacker) [inline packages/mate_desktop.rb:42]: `run_tests`?

File: packages/caja.rb (hunk 1 above)
Comment (satmandu) [reply packages/caja.rb:56]: Wouldn't it make sense for including `no_run_tests` as needed?

Comment (uberhacker) [reply packages/caja.rb:56]: No, that is why I added `run_tests`.  So it wouldn't run tests automatically but only when requested.

Comment (uberhacker) [reply packages/caja.rb:56]: Well, that would force a test run for every build.  But what about this case where there are no tests available?

**Review by uberhacker — APPROVED** [review]
I'll go ahead and merge this since tests are not available.

File: packages/caja.rb (hunk 1 above)
Comment (satmandu) [reply packages/caja.rb:56]: I guess, the question is... Which is more common?

If `make test` is more often available than not, maybe that should be the default for a build macro like build systems?

**Pull request merged**